new-lamassu-admin/src/pages/ServerLogs.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from 'react'
+import { get, concat, uniq } from 'lodash/fp'
+import moment from 'moment'
+import useAxios from '@use-hooks/axios'
+
+import { Table, TableHead, TableRow, TableHeader, TableBody, TableCell } from '../components/table'
+import { Select } from '../components/inputs'
+import Uptime from '../components/Uptime'
+import LogPageHeader from '../components/LogPageHeader'
+import { makeStyles } from '@material-ui/core'
+import typographyStyles from '../components/typography/styles'
+
+import { zircon, comet, white, fontSecondary } from '../styling/variables'
+
+const { regularLabel } = typographyStyles
+
+const styles = {
+  titleWrapper: {
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  serverVersion: {
+    extend: regularLabel,
+    color: comet,
+    margin: 'auto 0 auto 0'
+  },
+  headerLine2: {
+    height: 60,
+    display: 'flex',
+    justifyContent: 'space-between',
+    marginBottom: 24
+  },
+  uptimeContainer: {
+    margin: 'auto 0 auto 0'
+  }
+}
+
+const useStyles = makeStyles(styles)
+
+const SHOW_ALL = 'Show all'
+
+const formatDate = date => {
+  return moment(date).format('YYYY-MM-DD HH:mm')
+}
+
+const Logs = () => {
+  const [machines, setMachines] = useState(null)
+  const [selected, setSelected] = useState(null)
+  const [saveMessage, setSaveMessage] = useState(null)
+  const [logLevel, setLogLevel] = useState(SHOW_ALL)
+  const [version, setVersion] = useState(null)
+  const [processStates, setProcessStates] = useState(null)
+
+  const classes = useStyles()
+
+  useAxios({
+    url: 'http://localhost:8070/api/version',
+    method: 'GET',
+    trigger: [],
+    customHandler: (err, res) => {
+      if (err) return
+      if (res) {
+        setVersion(res.data)
+      }
+    }
+  })
+
+  useAxios({
+    url: 'http://localhost:8070/api/uptimes',
+    method: 'GET',
+    trigger: [],
+    customHandler: (err, res) => {
+      if (err) return
+      if (res) {
+        setProcessStates(res.data)
+        console.log(res.data)
+      }
+    }
+  })
+
+  useAxios({
+    url: 'http://localhost:8070/api/machines',
+    method: 'GET',
+    trigger: [],
+    customHandler: (err, res) => {
+      if (err) return
+      if (res) {
+        setMachines(res.data.machines)
+        setSelected(get('data.machines[0]')(res))
+      }
+    }
+  })
+
+  const { response: logsResponse } = useAxios({
+    url: `http://localhost:8070/api/logs/${get('deviceId', selected)}`,
+    method: 'GET',
+    trigger: selected,
+    forceDispatchEffect: () => !!selected,
+    customHandler: () => {
+      setSaveMessage('')
+    }
+  })
+
+  const { loading, reFetch: sendSnapshot } = useAxios({
+    url: `http://localhost:8070/api/support_logs?deviceId=${get('deviceId')(selected)}`,
+    method: 'POST',
+    customHandler: (err, res) => {
+      if (err) {
+        setSaveMessage('Failure saving snapshot')
+        throw err
+      }
+      setSaveMessage('✓ Saved latest snapshot')
+    }
+  })
```

Comment:
This looks like the one from machine, we don't have device_id for this one, and the support logs db is different for lamassu-server logs